Name: No Network Time Zone Error
Version: 1.3.1-1
Author: Jason Robitaille
Description: Under some circumstances, even with "network time" and "network time zone" disables, you'll still get an error at reboot about the network time zone be invalid.  This patch gets rid of that error message.  (MIT license)

--- .orig/usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
+++ /usr/lib/luna/system/luna-systemui/app/controllers/bar-assistant.js
@@ -180,8 +180,8 @@
 		return;
 	this.startClock();	
 	
-	if(payload.NITZValid != undefined && payload.NITZValid === false)
-		this.showTimezoneErrorAlert();
+	//if(payload.NITZValid != undefined && payload.NITZValid === false)
+	//	this.showTimezoneErrorAlert();
 },
 
 registerTimeFormat:function(){
